citus_10_pg_12_5.dockerfile

```
@@ -0,0 +1,40 @@
+FROM postgres:12.5
```

let's use newest version of pg12

```suggestion
FROM postgres:12.6
```